acbuild/acbuild.go

```diff
@@ -140,7 +144,45 @@ GLOBAL OPTIONS:
 // terminator.
 func runWrapper(cf func(cmd *cobra.Command, args []string) (exit int)) func(cmd *cobra.Command, args []string) {
 	return func(cmd *cobra.Command, args []string) {
+		if aciToModify != "" {
+			contextualCommands := []string{"begin", "end", "abort"}
+			for _, cc := range contextualCommands {
+				command := strings.Split(cmd.Use, " ")[0]
+				if command == cc {
+					stderr("Can't use the --modify flag with %s.", command)
+					cmdExitCode = 1
+					return
+				}
+			}
+
+			var err error
+			contextpath, err = ioutil.TempDir("", "acbuild-")
+			if err != nil {
+				stderr("%v", err)
```

Comment:
Nope, the first argument to stderr must be a string.

https://play.golang.org/p/2CvjwRqRwF